echopype/preprocess/noise_est.py

```diff
@@ -113,13 +113,22 @@ def remove_noise(self, noise_max=None, SNR_threshold=3):
         )  # other=-999 (from paper)
 
         # Assemble output dataset
-        self.ds_Sv["Sv_corrected"] = Sv_corr
-        self.ds_Sv["Sv_noise"] = self.Sv_noise
-        self.ds_Sv = self.ds_Sv.assign_attrs(
-            {
+        def add_attrs(sv_type, da):
+            da.attrs = {
+                "long_name": f"Volume backscattering strength, {sv_type} (Sv re 1 m-1)",
+                "units": "dB",
+                "actual_range": [
+                    round(float(da.min().values), 2),
+                    round(float(da.max().values), 2),
+                ],
                 "noise_ping_num": self.ping_num,
                 "noise_range_sample_num": self.range_sample_num,
                 "SNR_threshold": SNR_threshold,
                 "noise_max": noise_max,
             }
-        )
+
+        self.ds_Sv["Sv_noise"] = self.Sv_noise
+        add_attrs("noise", self.ds_Sv["Sv_noise"])
+
+        self.ds_Sv["Sv_corrected"] = Sv_corr
+        add_attrs("corrected", self.ds_Sv["Sv_corrected"])
```

Comment:
Should we also add like what you have for `compute_MVBS`?
```suggestion
        add_attrs("corrected", self.ds_Sv["Sv_corrected"])
        
        prov_dict = echopype_prov_attrs(process_type="processing", source_files=None)
        prov_dict["processing_function"] = "preprocess.remove_noise"
        ds_Sv = ds_Sv.assign_attrs(prov_dict)
```

Reply:
I have the same `source_files=None` thoughts as in the `ds_MVBS` section.

Reply:
> Should we also add like what you have for compute_MVBS?

Heh. That provenance info is already being added, in [`preprocess/api.py`, line 283](
https://github.com/OSOceanAcoustics/echopype/pull/621/files/c1ebfb36478b5c7a3f52a1f817a00abd9d23a1af#diff-2848209b968526da06d5475e3df089d5a7138b944681119fbe9f64f9aad81603R283). ie, downstream. This was trickier than for `compute_MVBS`, b/c there's an external function (`preprocess/noise_est.py`) doing the heavy lifting.

Reply:
Ah! I missed that. Thanks!

Reply:
> I have the same source_files=None thoughts as in the ds_MVBS section.

See my reply there